react/features/whiteboard/actions.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable arrow-body-style */
```

Comment:
Please don't disable rules just because you prefer a different style. Consistency is valuable.